impl/src/main/java/org/ehcache/config/event/DefaultCacheEventListenerConfiguration.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.config.event;
+
+import org.ehcache.event.CacheEventListener;
+import org.ehcache.event.CacheEventListenerConfiguration;
+import org.ehcache.event.CacheEventListenerFactory;
+import org.ehcache.event.EventFiring;
+import org.ehcache.event.EventOrdering;
+import org.ehcache.event.EventType;
+import org.ehcache.internal.classes.ClassInstanceProviderConfig;
+
+import java.util.EnumSet;
+
+/**
+ * Created by rism on 3/2/15.
```

Comment:
Can you get rid of that template?
It adds nothing - the same info exists in the version control system.
However, adding some useful javadoc on the type users may end seeing is a good idea.